index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <!-- <div class="container"> -->
+      <nav>
+        <div class="logo">
+          <a href="#" style="text-decoration: none;"><h4>My logo</h4></a>
+        </div>
+        <ul class="nav_links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">about</a></li>
+          <li><a href="#">contact</a></li>
+          <i class="fa-solid fa-bars"></i>
+        </ul>
+      </nav></header>
+      <section class="present">
+        <h1>I’m Drissa, Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <h3>Let's connect</h3>
+        <ul class="social">
+        <li>  <a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+        <li>  <a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+        <li>  <a href="#"><i class="fa-brands fa-mastodon"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+        </ul>
+      </section>
+    <!-- </div> -->
+    
+
+    <!-- wor section for desktop -->
+
+    <section class="desk_work">
+        <div class="item">
+            <img src="./images/desk2_Snapshoot Portfolio.png" alt="img1" class="projet_img">
+            <div class="info_projet">
+                <h2>Tonic</h2>
+                <div class="type">
+                    <p class="client">CANOPY</p>
+                    <img src="./images/circle.png" alt="">
+                    <p class="service">Back End Dev</p>
+                    <img src="./images/circle.png" alt="">
+                    <p class="service">2015</p>
+                </div>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="languages">
+                    <p>html</p>
+                    <p>css</p>
+                    <p>javaScript</p>
+                </div>
```

Comment:
- [ ] The senior code reviewer @nicupop729 mentioned this comment to use the `ul` tag as a mandatory change, please make sure you fix it before your project gets approved.

Reply:
@joskalenda fixed it